chroma_core/management/commands/print-settings.py

```diff
@@ -16,6 +17,21 @@ class Command(BaseCommand):
 
     help = """Prints out selected settings in env variable format"""
 
+    def _load_config(self, config_file):
+        if os.path.exists(config_file):
+            file = open(config_file, "r")
```

Comment:
use the `with` context handler for opening files. ref: https://www.pythonforbeginners.com/files/with-statement-in-python